commitlint.config.js

```diff
@@ -13,7 +13,8 @@ const applyCustomScope = () => {
     'guide',
     'tech-snacks',
     'lint',
-    'scaffolding'
+    'scaffolding',
+    'shared-styles'
```

Comment:
How about renaming this to 'shared' to better match the directory name?

Reply:
My thought is to make the scope as generic as possible while referring to a tangible directory name. The commit `chore(shared): remove unused tooltip class` is descriptive enough, and the scope is reusable for different contexts that can be described by the subject. Really though, it's personal preference; the final choice is yours.

The PR is good whether or not you keep this change 👍

Reply:
Oh, I didn't realize you were trying to base the scopes on the folder names! I think thats a pretty good idea. In that case we are missing some things. Maybe I'll make another PR for that and see what that looks like.